.gitignore

```diff
@@ -13,6 +13,14 @@
 .DS_Store
 xcuserdata
 /Awful.xcworkspace/xcshareddata/Awful.xccheckout
+*.backup
```

Comment:
idk what this is, is it generated by xcode somehow? If it's an editor or tool you're using, might be better added to a global gitignore on your computer.